packages/sftp-client/index.js

```diff
@@ -119,7 +119,8 @@ class SftpClient {
       Bucket: bucket,
       Key: key,
       Body: pass,
-      ContentType: lookupMimeType(key)
+      // eslint-disable-next-line unicorn/no-null
+      ContentType: mime.lookup(key) || null
```

Comment:
Not sure why that was done. It's `undefined` now. Incidentally, I'm explicitly setting a value because `mime.lookup()` returns `false` if a type could not be found, which is not what we want.